CMakeLists.txt

```diff
@@ -271,9 +271,20 @@ endif()
 # If it's preexisting, leave it alone.
 include(InstallHelpers)
 
+# For $CMAKE_INSTALL_{DATAROOT,MAN, ...}DIR
```

Comment:
> Maybe ${CMAKE_INSTALL_*_DIR}?

I'll do `$CMAKE_INSTALL_*`: no need for the `{}` markup IMO.

> And maybe move it up above the InstallHelpers inclusion, because that already uses these variables (although currently only within a function)?

Sure, I'll do that. I didn't understand why you wanted to do that before, but I I understand after the above explanation.